src/Compilers/CSharp/Test/Semantic/SourceGeneration/GeneratorDriverTests.cs

```diff
@@ -1274,25 +1271,22 @@ class C { }
             Assert.Equal(1, initCount);
             Assert.Equal(1, executeCount);
 
-            // ran the incremental generator if in preview
-            Assert.Equal(langVer == LanguageVersion.Preview ? 1 : 0, incrementalInitCount);
+            // ran the incremental generator if supported
```

Comment:
Comment seems out-of-date. Incremental generators are supported regardless of langver now